_commands/tools/time.md

```diff
@@ -0,0 +1,123 @@
+---
+---
+
+time
+---
+`time` prints out the total time elapsed, the time used by system overhead and the time used to execute the utility.
+	
+### General Form:
+
+~~~ bash
+$ /usr/bin/time [ -apqvV ] [ -f FORMAT ] [ -o FILE ] [ --append ] [ --verbose ] [ --quiet ] [ --portability ]
```

Comment:
I don't like this; I think it's intimidating and is the root of what made Pat and I say this looks like a man page excerpt.